### PR TITLE
TE-1864 Add character limit to inputs

### PR DESCRIPTION
**Pull request by red-runner**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-1864)

### What **one** thing does this PR do?
Implements a `maxCharacters` prop for the `TextArea` and `TextInput` component; then, updates the components listed in the ticket with the required values. 

### TextArea behaviour 
textarea

### TextInput behaviour
textinput

### CODE REVIEW

**Comment by red-runner** [comment]
> Shouldn't we control the max characters in templates-ssr and just expose the maxCharacter prop from the lodgify ui components api?

I think would make more sense. The only thing is that  it would be great to address this kind of matters in planning/grooming. The ticket specifies to do all the work in lodgify-ui 😭

File: src/components/general-widgets/CallMeBack/constants.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const EMAIL_MAX_LENGTH = 255;
+export const NAME_MAX_LENGTH = 80;
+export const NOTES_MAX_LENGTH = 800;
+export const PHONE_MAX_LENGTH = 15;
```
Comment (marcdavi-es) [inline src/components/general-widgets/CallMeBack/constants.js]: Let's call these `*_MAX_CHARACTERS` for consistency

File: src/components/general-widgets/Contact/constants.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const COMMENT_MAX_LENGTH = 800;
+export const EMAIL_MAX_LENGTH = 255;
+export const NAME_MAX_LENGTH = 80;
+export const PHONE_MAX_LENGTH = 15;
```
Comment (marcdavi-es) [inline src/components/general-widgets/Contact/constants.js]: Same as https://github.com/lodgify/lodgify-ui/pull/569/commits/2b558fff786a27558ef58e025eccf0f669e7a95b#r263334761

File: src/components/general-widgets/OwnerLogin/constants.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const EMAIL_MAX_LENGTH = 255;
+export const PASSWORD_MAX_LENGTH = 255;
```
Comment (marcdavi-es) [inline src/components/general-widgets/OwnerLogin/constants.js]: Same as https://github.com/lodgify/lodgify-ui/pull/569/commits/2b558fff786a27558ef58e025eccf0f669e7a95b#r263334761

File: src/components/general-widgets/OwnerSignUp/constants.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+export const EMAIL_MAX_LENGTH = 255;
+export const FIRST_NAME_MAX_LENGTH = 80;
+export const LAST_NAME_MAX_LENGTH = 80;
```
Comment (marcdavi-es) [inline src/components/general-widgets/OwnerSignUp/constants.js]: Same as https://github.com/lodgify/lodgify-ui/pull/569/commits/2b558fff786a27558ef58e025eccf0f669e7a95b#r263334761

File: src/components/property-page-widgets/Reviews/constants.js [hunk 5]
```diff
@@ -13,3 +13,9 @@ const NUMBER_OF_PLACEHOLDERS = 3;
 export const PLACEHOLDERS = Array(NUMBER_OF_PLACEHOLDERS).fill(
   PLACEHOLDER_DATA
 );
+
+export const COMMENT_MAX_LENGTH = 800;
+export const EMAIL_MAX_LENGTH = 255;
+export const LOCATION_MAX_LENGTH = 255;
+export const NAME_MAX_LENGTH = 80;
+export const TITLE_MAX_LENGTH = 30;
```
Comment (marcdavi-es) [inline src/components/property-page-widgets/Reviews/constants.js]: https://github.com/lodgify/lodgify-ui/pull/569/commits/2b558fff786a27558ef58e025eccf0f669e7a95b#r263334761

**Pull request merged**